C++14/dp/lps_in_O(n).cpp

```diff
@@ -0,0 +1,55 @@
+//Returns Longest Palindrome Subsequence length. 
+//Time Complexity O(n).
```

Comment:
Please mention the space complexity as well.

Reply:
Done.

Reply:
Done with all the comments.